components/kserve/kserve.go

```diff
@@ -54,6 +54,8 @@ type Kserve struct {
 	// This field is optional. If no default deployment mode is specified, Kserve will use Serverless mode.
 	// +kubebuilder:validation:Enum=Serverless;RawDeployment
 	DefaultDeploymentMode DefaultDeploymentMode `json:"defaultDeploymentMode,omitempty"`
+	// Configures and enables Nvidida's NIM integration
```

Comment:
is the  intention only to enable or we will later add more config to "configurate"?

Reply:
@zdtsw the configuration part will be done from the [odh-model-controller](https://github.com/opendatahub-io/odh-model-controller/tree/main) based on the enabled flag

Reply:
:heavy_check_mark: 
thats what i was thinking, but not 100% sure about it.